src/inspector/models/general/appearance/appearancesettingsmodel.cpp

```diff
@@ -103,16 +103,48 @@ void AppearanceSettingsModel::loadOffsets()
     });
 }
 
-void AppearanceSettingsModel::pushBackInOrder()
+std::vector<Ms::EngravingItem*> AppearanceSettingsModel::getAllElementsInPage()
+{
+    requestElements();
+    Ms::EngravingItem* element = m_elementList.first();
+    Ms::Page* page = toPage(element->findAncestor(Ms::ElementType::PAGE));
+    return page->elements();
+}
+
+void AppearanceSettingsModel::pushBackwardsInOrder()
 {
     m_arrangeOrder->setValue(m_arrangeOrder->value().toInt() - REARRANGE_ORDER_STEP);
 }
 
-void AppearanceSettingsModel::pushFrontInOrder()
+void AppearanceSettingsModel::pushForwardsInOrder()
 {
     m_arrangeOrder->setValue(m_arrangeOrder->value().toInt() + REARRANGE_ORDER_STEP);
 }
 
+void AppearanceSettingsModel::pushToBackInOrder()
```

Comment:
FWIW, this seems to be the exact same approach that would also be needed to fix the existing Forwards/Backwards buttons, which basically do almost nothing now because they only move by REARRANGE_ORDER_STEP rather than finding the previous/next element and pushing past that.  So currently they require you to click Forwards literally dozens of times before you see anything happen.

I mention this here because ideally I think both of these issues should be addressed to together so the code is designed well to handle both commands with a minimum of duplication.

Reply:
Thanks, that's an interesting point. I'll look into that to see if there's an elegant solution for both problems.

Reply:
I've updated the pull request to support that feature as well :)